pkg/gatewayconfigurationserver/gatewayconfigurationserver.go

```diff
@@ -0,0 +1,113 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gatewayconfigurationserver
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/gogo/protobuf/types"
+	echo "github.com/labstack/echo/v4"
+	bs_cups "go.thethings.network/lorawan-stack/pkg/basicstation/cups"
```

Comment:
I don't think underscores in imports are what we typically do

Reply:
There isn't really a guideline for that. The gRPC plugins/middlwares we use also have underscores (even in their package name). Using `bscups` is fine here.